__tests__/tests/utilities/color-helpers.js

```diff
@@ -0,0 +1,33 @@
+import { getFontColorByBackground } from 'utils';
+
+describe('Color Helpers', () => {
+  it('should return a black font color for the light background #1', () => {
```

Comment:
Could you say something like `'should return a black font color for the light yellow background'` and apply that format to the rest of these tests?